specification/communication/data-plane/role/preview/2022-01-01-preview/exmaples/CreateRole.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
There's a typo in the folder path unfortunately. exmaples => examples

Reply:
Fixed.